tests/test_sort.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+import numpy as np
+import warnings
+from strax.sort_enforcement import SortingError, stable_sort, stable_argsort
+
+
+class TestSortEnforcement(unittest.TestCase):
+    def setUp(self):
+        """Set up test fixtures before each test method."""
+        self.arr = np.array([3, 1, 4, 1, 5, 9, 2, 6])
+        # Store expected sorted array and indices for comparison
+        self.expected_sorted = np.array([1, 1, 2, 3, 4, 5, 6, 9])
+        self.expected_argsort = np.array([1, 3, 6, 0, 2, 4, 7, 5])
+
+    def test_explicit_stable_sort(self):
+        """Test explicit stable_sort function (should not warn)"""
+        with warnings.catch_warnings():
+            warnings.simplefilter("error")  # Turn warnings into errors
+            sorted_arr = stable_sort(self.arr)
+            np.testing.assert_array_equal(sorted_arr, self.expected_sorted)
+
+    def test_explicit_stable_sort_argsort(self):
```

Comment:
What is `stable_sort_argsort`? Should be `stable_argsort`?

Reply:
fixed